scene/3d/path_3d.h

```diff
@@ -124,7 +124,7 @@ class PathFollow3D : public Node3D {
 
 	PackedStringArray get_configuration_warnings() const override;
 
-	void update_transform(bool p_immediate = false);
+	void update_transform(bool p_immediate = true);
```

Comment:
Seems like the `p_immediate` argument should be removed together with the deferred branch. I don't see any code explicitly calling `update_transform(false)`, and the only `update_transform(true)` is in `tests/scene/test_path_follow_3d.h` and can now be simplified.

This would also restore consistency with PathFollow2D.

Reply:
Bump. I could update the PR to remove it myself but it questions a lot of the changes added in #80233 that need to be reviewed.

The "deferred" mode referenced in multiple comments no longer exists as it's never used, and the need for `update_transform` that would just set the dirty flag before calling `_update_transform` is now put into question too.

Could you take a closer look and see what code should be changed as a consequence of this fix?